src/types.ts

```diff
@@ -90,6 +90,12 @@ export interface Options {
    */
   globs?: string | string[]
 
+  /**
+   * An array of glob patterns to exclude matches, This is an alternative way to use negative patterns.
+   * @default ['node_modules']
+   */
+  ignore?: string[]
```

Comment:
Maybe `globsExclude`? Actually, you can pass `!xxx` to `globs` to exclude some iirc

Reply:
of course, but we also need to change code.

if I set 
```js
options.dirs = [
        'src/components',
        '!src/components/AAA.vue'
      ]
```
now, they will be transformed to
```js
'C:/Users/cuilu/Desktop/issues/unplugin-components-601/src/components/**/*.vue'
'C:/Users/cuilu/Desktop/issues/unplugin-components-601/!src/components/AAA.vue/**/*.vue'
```

Reply:
if user just use options.globs, globsExclude is ok, you can make a decision.

Reply:
Having `globsExclude` is ok

Reply:
should we change the code to transform dirs?
or do nothing, let user use globs.

Reply:
@antfu 🤣🤣🤣，options.globs also have problem

```js
globs: [
        'src/components/**/*.vue',
        '!src/components/**/AAA.vue',
      ]
```

globs also be converted.
```js
 [
'C:/Users/cuilu/Desktop/issues/unplugin-components-601/src/components/**/*.vue', 

'C:/Users/cuilu/Desktop/issues/unplugin-components-601/!src/components/**/AAA.vue'
]
```

![image](https://github.com/antfu/unplugin-vue-components/assets/46661044/88d120f4-b888-490e-850d-897c62fd8123)

Reply:
should i change options.dirs and options.globs,  or only change options.globs to make globsExclude work?

Reply:
We could check if it starts with `!`, if so, remove it and do the resolve and than adding back the `!`

Reply:
OK